test/e2e/leaderelection.go

```diff
@@ -76,17 +76,19 @@ var _ = Describe("Leader Elector", func() {
 })
 
 func spawnControllerManagerProcess(tl common.TestLogger, kubeConfigPath, namespace string) (*exec.Cmd, io.ReadCloser, error) {
-	// Get the directory of the current executable
-	_, filename, _, _ := runtime.Caller(0)
-	managedPath := filepath.Dir(filename)
-	confFile, err := filepath.Abs(fmt.Sprintf("%s/../../config/kubefedconfig.yaml", managedPath))
+	kubefedconfigAsset := common.MustAsset("config/kubefedconfig.yaml")
```

Comment:
Please use a more specific asset name like 'leaderelectionconfig.yaml' to ensure that future maintainers don't reuse it for other purposes accidentally.

(No action required) I'm not sure it's desirable to suffix the name of the variable with 'Assert'.  Once loaded, it's just a byte array containing yaml, and there is no need to indicate where it was loaded from (e.g. I wouldn't expect a variable containing a byte array loaded from disk to be named 'byteArrayFromDisk').

Reply:
Also, `kubeFedConfig`?